components/automate-cli/cmd/chef-automate/iam.go

```diff
@@ -420,7 +420,7 @@ func runCreateTokenCmd(cmd *cobra.Command, args []string) error {
 		Projects: []string{},
 	})
 	if err != nil {
-		return status.Wrap(err, status.APIError, "Failed to generate new admin token.")
+		return status.Wrap(err, status.APIError, "Failed to generate new token.")
```

Comment:
Good catch! However, if that changes I suspect the constant `adminTokenIAMPreconditionError` should also change.

Reply:
I checked, that error only triggers when you pass the `--admin` flag on v1. otherwise `chef-automate iam token create` works the same on v1 and v2